Uno.Gallery/Platforms/WebAssembly/Program.cs

```diff
@@ -17,8 +17,7 @@ static int Main(string[] args)
 		{
 			// Ask the browser to preload these fonts to avoid relayouting content
 #if IS_WASM_SKIA
-			// Disabled https://github.com/unoplatform/uno-private/issues/777
-			// FontFamilyHelper.PreloadAsync("Symbols", FontWeights.Normal, Windows.UI.Text.FontStretch.Normal, Windows.UI.Text.FontStyle.Normal);
+			await FontFamilyHelper.PreloadAsync("ms-appx:///Uno.Fonts.Roboto/Fonts/Roboto-Regular.ttf#Roboto", FontWeights.Normal, Windows.UI.Text.FontStretch.Normal, Windows.UI.Text.FontStyle.Normal);
```

Comment:
Roboto is already set as the DefaultTextFont, which is the font that we preload. So this shouldn't be needed and it seems like it's already working, it was just a problem with canary builds I think.

Reply:
Yes, agreed, this is not happening anymore.